From 221822c11d8aca125e36bdcd26ac842835f90be2 Mon Sep 17 00:00:00 2001
From: Ikey Doherty <michael.i.doherty@intel.com>
Date: Thu, 28 Jan 2016 13:50:09 +0000
Subject: [PATCH] Enforce a 3.22 gnome version to prevent autogen issues

Signed-off-by: Ikey Doherty <michael.i.doherty@intel.com>
---
 autogen.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/autogen.sh b/autogen.sh
index 0e36e12..d58829d 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -10,4 +10,4 @@ mkdir -p m4
 autoreconf --verbose --force --install || exit $?
 
 cd "$cwd"
-"$srcdir/configure" $@
+"$srcdir/configure" $@ --with-gnome=3.22
-- 
2.12.2

